docs/ldml/tr35-personNames.md

```diff
@@ -41,26 +41,929 @@ The LDML specification is divided into the following parts:
+
+|              | EBNF           | Comments    |
+| ------------ | -------------- | ----------- |
+| namePattern  | = literal?<br>( modField  literal? )+; | Two literals cannot be adjacent |
+| modField     | = '{' field modifierList? '}';         | Parts of a name, described below |
+| field        | = 'prefix'<br>&#124; 'given'<br>&#124; 'given2'<br>&#124; 'surname'<br>&#124; 'surname2'<br>&#124; 'suffix' ; | Parts of a name, described below |
+| modifierList | = '-informal'?<br>( '-allCaps' &#124; ‘-initialCap' )?<br>( '-initial' &#124; '-monogram' )?<br>( '-prefix' &#124; '-core' )? | Optional modifiers that can be applied to name parts. Note that some modifiers are exclusive: only `prefix` or `core`, only `initial` or `monogram`, only `allCaps` or `initialCap`. |
+| literal      | = codepoint+ ; | One or more Unicode codepoints. |
+
+### 5.1 <a name="5-1-fields" href="#5-1-fields">Fields</a>
+
+The Person Name formatting data assumes that the name data to be formatted consists of the fields in the table below. All of the fields may contain multiple words. Field IDs are lowercase ASCII alphanumeric, and start with an alphabetic character.
+
+When determining how a full name is to be placed into name fields, the data to be formatted should be organized functionally. That is, if a name part is on the dividing line between `given2` and `given`, the key feature is whether it would always occur with the rest of the given name. For example, in “_Mary Jean Smith_”, if _Mary_ never occurs without the “_Jean”_, then the given name should be “_Mary Jean_”. If “_Smith_” never occurs without the “_Jean_”, the `surname` should be “_Jean Smith_”. Otherwise, “_Jean_” would be the `given2` field. 
+
+For example, a patronymic would be treated as a `given2` name in most slavic languages. 
+
+In some cultures, two surnames are used to indicate the paternal and maternal family names or generational names indicating father, grandfather. The `surname2` field is used to indicate this. The CLDR PersonName formatting data assumes that if a PersonName object to be formatted does not have two surnames, then the `surname2` field is not populated. (That is, no pattern should have a `surname2` field without a surname field.) Order of fields in a pattern can vary arbitrarily by locale.
+
+In most cultures, there is a concept of nickname or preferred name, which is used in informal settings or sometimes to represent a “public” or “stage name”. The nickname or preferred name may be submitted as a separate PersonName object to be formatted, or included with a modifier such as `given-informal`. 
+
+| Field      | Description |
+| ---------- | ----------- |
+| `prefix`   | Typically a title, honorific, or generational qualifier.<br>Example: ‘Ms.’, ‘Mr.’, ’Dr’, ‘President’<br><br>Note that CLDR PersonaName formats data does not define regional or locale-specific lists of prefixes, honorifics, or titles such as “Mr”, “Ms”, “Mx”, “Prof”, “Jr”, etc. |
```

Comment:
Add: The values are as supplied by the name object, via the PersonName data interface.

Reply:
Done (in header)